### DISCUSSION TITLE
earData xsd taxPosition eleme

### DISCUSSION THREAD

**Discussion by bmatyi94** [nav-gov-hu/eVAT#30] [Kérdések és válaszok - Gépi interfész] [answered]
Sziasztok!

Az earData xsd-ben azt látom hogy a taxPosition elem 2-szer is megadásra kerülhet adott taxInformation elemen belül. Milyen olyan eset van ahol egy standard adókódhoz egyszerre tartozhat fizetendő és levonható pozíció is? Az standard adókódoknál úgy láttam hogy a fizetendő oldalhoz tartozó adókód más mint a levonható oldalhoz tartozó adókód. Például egy közösségen belüli termékbeszerzés esetében.

Köszönöm előre is a válaszotokat.

**Comment by NTCA-HM** [comment, marked as answer]
Kedves @bmatyi94 !

Adott bizonylathoz több standard adókód tartozhat. Bevallási pozíciókból egy választható az adott taxPosition csomópontnál(). Értékei: fizetendő (positionType = PAYABLE) és/vagy levonható (positionType = DEDUCTIBLE) vagy (positionType = OTHER) Még megvizsgáljuk, hogy ezt bővítjük még egy elemmel az egyértelműség kedvéért. Amennyiben a levonási jog feltételei adottak, akkor több esetben is előfordulhat, hogy a fizetendő és a levonható adóra vonatkozó bevallás sort is ki kell tölteni:
o	a közösségi termékbeszerzés
o	közösségi szolgáltatás igénybevétele
o	fordított áfás beszerzés vagy szolgáltatás igénybevétele esetén
o	önadózással bevallott termékimport
Illetve, ha mindkettőre vonatkozik akkor 1x kell szerepeltetni. Adókód alapján, most ezek az oldalak külön szerepelnek, de megnézzük, hogy összevont adókód alkalmazható-e.
